packages/charrua-core/charrua-core.0.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mirage/charrua-core/releases/download/v0.5/charrua-core-0.5.tbz"
```

Comment:
@haesbaert in theory yes, but the `v0.5.tar.gz` is generated (in unknown ways, with unknown content) by GitHub.  The `charrua-core-0.5.tbz` is generated in a reproducible way (with watermarks (such as `%%VERSION%%` replaced by the actual version), not containing any user id information, etc.) by `topkg-care` when doing the release, and then uploaded to GitHub (and IMHO preferable over GitHubs magic).  See http://erratique.ch/software/topkg/doc/Topkg.Pkg.html#VALdistrib